alma/views/templates/admin/_configure/helpers/form/form.tpl

```diff
@@ -53,6 +53,14 @@
                 {/strip}
             </div>
         {/foreach}
+    {else if $input.type == 'secret'}
+        {assign var='value_text' value=$fields_value[$input.name]}
+        <input type="password"
+            name="{$input.name}"
+            size="{$input.size}"
+            class="{if isset($input.class)}{$input.class}{/if}"
+            value="{if !empty($value_text)}*****{/if}"
```

Comment:
Can you use ApiHelper OBSCUR const in .TPL file?

Reply:
@Francois-Gomis 
I used the placeholder key.
Tell me what do you think ?